package.json

```diff
@@ -4,6 +4,7 @@
   "engines": {
     "node": ">=14"
   },
+  "packageManager": "yarn@1.22.19",
```

Comment:
Doing this also solves this issue: https://github.com/mastodon/documentation/issues/931

Reply:
This should be in a separate PR.

I had this in my TODO list, but you can do it as well.

Various docs will need to be updated to use `corepack` instead of manually installing yarn.

Reply:
Docs already say to use corepack, but manually enable yarn classic, this just does it automatically — this was a gotcha for me when I was trying to test this out on an ubuntu box